docs/Components/messages.md

```diff
@@ -17,11 +17,13 @@ property object with the following properties:
 | Property | Description  | Type | Required |
 | -- | -- | --| -- |
 | id | A unique identifier for the message | String | Yes |
+| usesSharedStrings | Whether it uses the translation file shared between all messages (default: false) | Boolean | No |
```

Comment:
I don't want to blow the scope up too much, but it seems less than ideal for this to be a top-level config for the  message addons. Consider for example, that at some point we want to release an update with some non-trivial messages. Having this exist as a top-level config means that we couldn't re-use any of the messages (even the boring stuff like the title).

Is it possible to have an approach where the addon generation script checks for translations in both the common and addon-specific locations, and chooses the appropriate one (defaulting to strings in the addon-specific location in the event of a conflict)?

Reply:
I think I caused some confusion here - you can mix generic and one time strings in a message. Is that what you mean? I think the confusion is from my naming of `usesSharedStrings`, and perhaps this variable should be updated.

The `strings.yaml` file is meant to handle both reusable strings and one-time strings. While reusable strings will stay in there forever, you can also create some category for `specificMessage`, and then set `vpn.specific225UpdateMessage.uniqueBody` for the content but `vpn.commonStrings.updateTitle` for the title. (I tried to give examples both the instructions in `strings.yaml` and in the commented out stuff in `strings.yaml`, but perhaps that needs to be improved as well.)

In that sense, `usesSharedStrings` is poorly named, as it's really a boolean of "do you use the old method or the new method of strings?". If this explanation was helpful and the issue is the naming of `usesSharedStrings`, let me know and I'll update it.